lib/ansible/plugins/filter/core.py

```diff
@@ -292,6 +292,20 @@ def mandatory(a):
     return a
 
 
+def recursive_check_defined(item):
```

Comment:
this might be better housed in `template/__init__.py` or` template.py` rather than `filters/core.py`

Reply:
Moved to template/__init__.py